proto/observer/crosschain_flags.proto

```diff
@@ -13,6 +13,10 @@ message GasPriceIncreaseFlags {
     (gogoproto.stdduration) = true
   ];
   uint32 gasPriceIncreasePercent = 3;
+
+  // Maximum gas price increase in percent of the median gas price
+  // Default is used if 0
+  uint32 gasPriceIncreaseMax = 4;
```

Comment:
Do we differentiate EVM-chain from BTC-chain on default `gasPriceIncreaseMax`

Reply:
Right now `GasPriceIncreaseFlags` is general for all chains. This is why we use percentage, the `medianGasLimit` will be specific for each chains so the final max number will be different for each chains.

For Bitcoin, gas price increase is disabled.

Reply:
Sounds cool. Thanks.